keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/resource/ImageHandler.kt

```diff
@@ -104,7 +105,16 @@ class ImageHandler(
     val appVersion = AppVersion.parseName(resourceDiff.desired.appVersion)
     val packageName = appVersion.packageName
     val version = resourceDiff.desired.appVersion.substringAfter("$packageName-")
-    val artifact = igorService.getArtifact(packageName, version)
+    val artifact = Artifact.builder()
+      .type("DEB")
+      .customKind(false)
+      .name(resource.spec.artifactName)
+      .version(version)
+      .location("rocket")
+      .reference("/${packageName}_${version}_all.deb")
```

Comment:
Is this .deb file naming pattern something that’s conventional enough we can rely on/expect non-Netflix builds to do it? I ask because I have literally no idea how the Debian ecosystem works beyond what I know from Netflix.

Reply:
That’s a good question. I asked a similar one in chat today. The short answer is “I don’t know”, but I got feedback that the `_all` suffix in the package name is standard at Netflix. Also, I hard-coded `rocket` there for the `location` field because, in my understanding, that’s the only artifact source we’re supporting today, which is very Netflix-specific. I’d love for @emjburns to chime in though.

Reply:
The debian stuff we're doing right now is very netflix specific. That's a fine assumption for now I think. I wouldn't worry at this moment about generalizing any debian logic - we don't know of anyone else who wants to use it.